solver/exec.go

```diff
@@ -115,7 +117,13 @@ func (e *execOp) Run(ctx context.Context, inputs []Reference) ([]Reference, erro
 	defer stdout.Close()
 	defer stderr.Close()
 
-	if err := e.w.Exec(ctx, meta, root, mounts, nil, stdout, stderr); err != nil {
+	iface, err := network.CreateNetworkInterface("docker0")
+	if err != nil {
+		logrus.Debugf("Unable to create NetworkInterface: %v", err)
+		iface = nil
+	}
+
+	if err := e.w.Exec(ctx, meta, root, mounts, nil, stdout, stderr, iface); err != nil {
```

Comment:
Add it to worker constructor https://github.com/moby/buildkit/blob/d66501e254ff98c2f0ae06d53c73bad5643aa9b1/control/control_standalone.go#L41 not to the call.